src/AspNetCoreModuleV2/CommonLib/stdafx.h

```diff
@@ -31,14 +33,16 @@
 #include "application.h"
 #include "SRWLockWrapper.h"
 #include "environmentvariablehash.h"
+#include "fx_ver.h"
```

Comment:
You seem to include this header into h and cpp files themselves. Do you need it here?

Reply:
Same for GlobalVersionUtility